libdnf/base/goal.cpp

```diff
@@ -48,7 +48,7 @@ static libdnf::rpm::PackageQuery running_kernel_check_path(libdnf::rpm::PackageS
     if (access(fn.c_str(), F_OK)) {
         // TODO(jmracek) Report g_debug("running_kernel_check_path(): no matching file: %s.", fn);
     }
-    libdnf::rpm::PackageQuery q(sack.get_weak_ptr(), libdnf::rpm::PackageQuery::InitFlags::IGNORE_EXCLUDES);
+    libdnf::rpm::PackageQuery q(sack.get_base(), libdnf::rpm::PackageQuery::InitFlags::IGNORE_EXCLUDES);
```

Comment:
If `running_kernel_check_path` is public I recommend to use Base directly as an argument.

Reply:
Changed. Base is argument now.